### PR TITLE
log workflow instance would be more useful

### PR DESCRIPTION
**Pull request by honnix**

### CODE REVIEW

File: styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java [hunk 1]
```diff
@@ -68,6 +68,8 @@ public TerminationLogMessage(
   }
 
   private static Optional<Integer> getExitCode(Pod pod, ContainerStatus status, Stats stats) {
+    final String workflowInstance = pod.getMetadata().getAnnotations()
+        .getOrDefault(KubernetesDockerRunner.STYX_WORKFLOW_INSTANCE_ANNOTATION, "N/A");
```
Comment (danielnorberg) [inline styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java]: Could also pass in the workflow id from the caller?

Comment (honnix) [inline styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java]: `pod` is used for other purpose in this method however. do we really want to add another param as signature?

**Comment by honnix** [comment]
@danielnorberg how about this?

File: styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java [hunk 2]
```diff
@@ -175,7 +180,10 @@ public TerminationLogMessage(
         exited = true;
         exitCode = pod.getStatus().getContainerStatuses().stream()
             .filter(IS_STYX_CONTAINER)
-            .map(cs -> getExitCode(pod, cs, stats))
+            .map(cs -> getExitCodeIfValid(
+                workflowInstance.toKey(),
+                pod.getMetadata().getAnnotations().get(DOCKER_TERMINATION_LOGGING_ANNOTATION),
```
Comment (danielnorberg) [inline styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java]: Could this stay inside `getExitCodeIfValid`?

Comment (honnix) [inline styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java]: So we keep `Pod` as the parameter? Sure. No strong opinion.

Comment (danielnorberg) [inline styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesPodEventTranslator.java]: ¯\\_(ツ)_/¯

Keeps the PR diff smaller

**Comment by danielnorberg** [comment]
👍

**Pull request merged**